src/main/java/org/codehaus/mojo/license/AbstractAddThirdPartyMojo.java

```diff
@@ -113,90 +113,152 @@
 
     /**
-     * A filter to exclude some GroupIds
-     * This is a regular expression that is applied to groupIds (not an ant pattern).
+     * A regular expression (not glob pattern) used as filter to exclude
+     * artifacts that have matching groupId. Match could be also partial for
+     * example {@code ^org\.}
+     * <p>
+     * This filter is applied on artifacts gathered as direct dependencies (and
+     * their transitive) of the projects in the reactor.
+     * <p>
+     * By default if an artifact is being excluded its transitive dependencies
+     * are not automatically. To drive this behaviour refer to
+     * {@link #includeTransitiveDependencies} and
+     * {@link #excludeTransitiveDependencies}.
      *
      * @since 1.1
      */
     @Parameter( property = "license.excludedGroups" )
     protected String excludedGroups;
 
     /**
-     * A filter to include only some GroupIds
-     * This is a regular expression applied to artifactIds.
+     * A regular expression (not glob pattern) used as filter to include
+     * only artifacts that have matching groupId.
+     * <p>
+     * This filter is applied on artifacts gathered as direct dependencies (and
+     * their transitive) of the projects in the reactor.
+     * <p>
+     * By default if an artifact is being included also its transitive dependencies
+     * are. To drive this behaviour refer to {@link #excludeTransitiveDependencies}.
      *
      * @since 1.1
-     */
-    @Parameter( property = "license.includedGroups" )
+     */    @Parameter( property = "license.includedGroups" )
     protected String includedGroups;
 
     /**
-     * A filter to exclude some ArtifactsIds
-     * This is a regular expression applied to artifactIds.
+     * A regular expression (not glob pattern) used as filter to exclude
+     * artifacts that have matching artifactId. Match could be also partial for
+     * example {@code ^org\.}
+     * <p>
+     * This filter is applied on artifacts gathered as direct dependencies (and
+     * their transitive) of the projects in the reactor.
+     * <p>
+     * By default if an artifact is being excluded its transitive dependencies
+     * are not automatically. To drive this behaviour refer to
+     * {@link #includeTransitiveDependencies} and
+     * {@link #excludeTransitiveDependencies}.
      *
      * @since 1.1
      */
     @Parameter( property = "license.excludedArtifacts" )
     protected String excludedArtifacts;
 
     /**
-     * A filter to include only some ArtifactsIds
-     * This is a regular expression applied to artifactIds.
+     * A regular expression (not glob pattern) used as filter to include
+     * only artifacts that have matching artifactId.
+     * <p>
+     * This filter is applied on artifacts gathered as direct dependencies (and
+     * their transitive) of the projects in the reactor.
+     * <p>
+     * By default if an artifact is being included also its transitive dependencies
```

Comment:
is included

Reply:
every "is being included" changed to "is included". The same also for "is being excluded"?